Emby.Drawing/SkiaEncoder.cs

```diff
@@ -270,17 +270,10 @@ internal static SKBitmap Decode(string path, bool forceCleanBitmap, IFileSystem
                     // create the bitmap
                     var bitmap = new SKBitmap(codec.Info.Width, codec.Info.Height, !requiresTransparencyHack);
 
-                    if (bitmap != null)
-                    {
-                        // decode
-                        codec.GetPixels(bitmap.Info, bitmap.GetPixels());
+                    // decode
+                    codec.GetPixels(bitmap.Info, bitmap.GetPixels());
```

Comment:
am I missing something, or shouldn't result of call to `codec.GetPixels()` be assigned somewhere?

Reply:
Wait, what does GetPixels modify? Is it void?

Reply:
No.
`Returns SkiaSharp.SKCodecResult.Success on success, or another value explaining the type of failure.`